java/kotlin-extractor2/src/main/kotlin/KotlinUsesExtractor.kt

```diff
@@ -252,32 +232,6 @@ open class KotlinUsesExtractor(
                 }
             }
 
-        // The Kotlin compiler internal representation of Outer<A, B>.Inner<C, D>.InnerInner<E,
-        // F>.someFunction<G, H>.LocalClass<I, J> is LocalClass<I, J, G, H, E, F, C, D, A, B>. This
-        // function returns [A, B, C, D, E, F, G, H, I, J].
-        private fun orderTypeArgsLeftToRight(
```

Comment:
Pleasingly this could die, because the Kotlin Analysis API represents this as `[[A, B], [C, D], [E, F], [G, H], [I, K]]` via the `KaClassType.qualifiers` property, which is a much more useful than the IR representation.